src/helpers/pressable-functions.php

```diff
@@ -191,9 +193,11 @@ function get_pressable_site_sftp_user_by_username( string $site_id, string $user
 function get_pressable_site_sftp_user_by_id( string $site_id, string $user_id ): ?object {
 	$sftp_users = get_pressable_site_sftp_users( $site_id );
 
-	foreach ( $sftp_users as $sftp_user ) {
-		if ( $user_id === (string) $sftp_user->id ) {
-			return $sftp_user;
+	if ( \is_array( $sftp_users ) ) {
```

Comment:
Rather than nesting the logic an extra level down, it may be simpler to just `if ( ! is_array ) { return null; }` and then do the subsequent looping assuming it is an array as we've passed that point?

Reply:
All the other helpers are as you suggested, so I tweaked this one to be like that too :)